Backends/RasterImage/port.lisp

```diff
@@ -30,7 +30,18 @@
 	   (slot-value port 'climi::sheet->mirror)))
 
 (defmethod destroy-port :before ((port raster-image-port))
- (%destroy-all-mirrors port))
+  (%destroy-all-mirrors port))
+
+(defmacro with-raster-image-port ((port-var server width height) &body body)
+  `(invoke-with-raster-image-port
+    (lambda (,port-var) ,@body) ,server ,width ,height))
+
+(defun invoke-with-raster-image-port (cont server width height)
+  (let* ((path (list server :width width :height height))
+         (port (find-port :server-path path)))
+    (unwind-protect
+         (funcall cont port)
+      (destroy-port port))))
```

Comment:
this should be a general utility since the same semantics are used for postscript and pdf as well. i.e `(with-temporary-port ((port-var path) &body body)`

Reply:
Done.